src/Courses/CourseCreation/TopicCreationModal.tsx

```diff
@@ -227,28 +235,44 @@ export const TopicCreationModal: React.FC<TopicCreationModalProps> = ({unitIndex
                     </Col>
                 </FormGroup>
                 <Row>
-                    <FormGroup as={Col} controlId='start'>
-                        <FormLabel>
-                            <h4>Start Date:</h4>
-                        </FormLabel>
-                        <FormControl 
-                            required
-                            type='date' 
-                            onChange={(e: any) => onTopicMetadataChange(e, 'startDate')}
-                            defaultValue={moment(topicMetadata?.startDate).format('YYYY-MM-DD')}
-                        />
-                    </FormGroup>
-                    <FormGroup as={Col} controlId='end'>
-                        <FormLabel>
-                            <h4>End Date:</h4>
-                        </FormLabel>
-                        <FormControl 
-                            required
-                            type='date' 
-                            onChange={(e: any) => onTopicMetadataChange(e, 'endDate')}
-                            defaultValue={moment(topicMetadata?.endDate).format('YYYY-MM-DD')}
-                        />
-                    </FormGroup>
+                    <MuiPickersUtilsProvider utils={MomentUtils}>
+                        <Col>
+                            <DateTimePicker 
+                                variant='inline'
+                                label='Start date'
+                                name={'start'}
+                                value={topicMetadata.startDate}
+                                onChange={() => {}}
+                                onAccept={(date: MaterialUiPickersDate) => {
+                                    if (!date) return;
+                                    const e = {target: {value: date.toDate()}};
+                                    onTopicMetadataChange(e, 'startDate');
+                                }}
+                                fullWidth={true}
+                                InputLabelProps={{shrink: false}}
+                                inputProps={{style: {textAlign: 'center'}}}
+                                defaultValue={moment(topicMetadata?.startDate).format('YYYY-MM-DD')}
```

Comment:
What happens if this is null? It uses current data?

Reply:
`moment(null)` uses the current datetime, yes.